<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="0" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="352612" started-at="2018-03-14T09:48:37Z" finished-at="2018-03-14T09:54:30Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="352612" started-at="2018-03-14T09:48:37Z" finished-at="2018-03-14T09:54:30Z">
      <class name="MobileApp.MobileApp.uber">
        <test-method status="FAIL" signature="setUp()[pri:0, instance:MobileApp.MobileApp.uber@7e0e6aa2]" name="setUp" is-config="true" duration-ms="352469" started-at="2018-03-14T15:18:37Z" finished-at="2018-03-14T15:24:30Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[An unknown server-side error occurred while processing the command. Original error: Error occured while starting App. Original error: Error executing adbExec. Original error: 'Command 'C\:\\Users\\salakhya\\AppData\\Local\\sdks\\platform-tools\\adb.exe -P 5037 -s ZY223ZKZ33 shell am start -W -n com.ubercab.eats/com.ubercab.eats.app.feature.launch.LobbyActivity -S' exited with code 1'; Stderr: 'java.lang.SecurityException: Permission Denial: starting Intent { flg=0x10000000 cmp=com.ubercab.eats/.app.feature.launch.LobbyActivity } from null (pid=17317, uid=2000) not exported from uid 10162at android.os.Parcel.readException(Parcel.java:1683)at android.os.Parcel.readException(Parcel.java:1636)at android.app.ActivityManagerProxy.startActivityAndWait(ActivityManagerNative.java:3266)at com.android.commands.am.Am.runStart(Am.java:634)at com.android.commands.am.Am.onRun(Am.java:390)at com.android.internal.os.BaseCommand.run(BaseCommand.java:51)at com.android.commands.am.Am.main(Am.java:122)at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:262)'; Code: '1' (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 349.53 secondsBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:14.666Z'System info: host: 'LAP-026', ip: '192.168.10.178', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_112'Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Error occured while starting App. Original error: Error executing adbExec. Original error: 'Command 'C\:\\Users\\salakhya\\AppData\\Local\\sdks\\platform-tools\\adb.exe -P 5037 -s ZY223ZKZ33 shell am start -W -n com.ubercab.eats/com.ubercab.eats.app.feature.launch.LobbyActivity -S' exited with code 1'; Stderr: 'java.lang.SecurityException: Permission Denial: starting Intent { flg=0x10000000 cmp=com.ubercab.eats/.app.feature.launch.LobbyActivity } from null (pid=17317, uid=2000) not exported from uid 10162at android.os.Parcel.readException(Parcel.java:1683)at android.os.Parcel.readException(Parcel.java:1636)at android.app.ActivityManagerProxy.startActivityAndWait(ActivityManagerNative.java:3266)at com.android.commands.am.Am.runStart(Am.java:634)at com.android.commands.am.Am.onRun(Am.java:390)at com.android.internal.os.BaseCommand.run(BaseCommand.java:51)at com.android.commands.am.Am.main(Am.java:122)at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:262)'; Code: '1' (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 349.53 secondsBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:14.666Z'System info: host: 'LAP-026', ip: '192.168.10.178', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_112'Driver info: driver.version: RemoteWebDriverat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)at java.lang.reflect.Constructor.newInstance(Unknown Source)at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)at org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$new$0(JsonWireProtocolResponse.java:53)at org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$getResponseFunction$2(JsonWireProtocolResponse.java:91)at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:123)at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyInto(Unknown Source)at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)at java.util.stream.AbstractPipeline.evaluate(Unknown Source)at java.util.stream.ReferencePipeline.findFirst(Unknown Source)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:126)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:219)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:142)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:155)at MobileApp.MobileApp.uber.setUp(uber.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.TestRunner.beforeRun(TestRunner.java:529)at org.testng.TestRunner.run(TestRunner.java:497)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="DeliveryLocation()[pri:0, instance:MobileApp.MobileApp.uber@7e0e6aa2]" name="DeliveryLocation" duration-ms="0" started-at="2018-03-14T15:24:30Z" finished-at="2018-03-14T15:24:30Z">
          <exception class="org.openqa.selenium.WebDriverException">
            <message>
              <![CDATA[An unknown server-side error occurred while processing the command. Original error: Error occured while starting App. Original error: Error executing adbExec. Original error: 'Command 'C\:\\Users\\salakhya\\AppData\\Local\\sdks\\platform-tools\\adb.exe -P 5037 -s ZY223ZKZ33 shell am start -W -n com.ubercab.eats/com.ubercab.eats.app.feature.launch.LobbyActivity -S' exited with code 1'; Stderr: 'java.lang.SecurityException: Permission Denial: starting Intent { flg=0x10000000 cmp=com.ubercab.eats/.app.feature.launch.LobbyActivity } from null (pid=17317, uid=2000) not exported from uid 10162at android.os.Parcel.readException(Parcel.java:1683)at android.os.Parcel.readException(Parcel.java:1636)at android.app.ActivityManagerProxy.startActivityAndWait(ActivityManagerNative.java:3266)at com.android.commands.am.Am.runStart(Am.java:634)at com.android.commands.am.Am.onRun(Am.java:390)at com.android.internal.os.BaseCommand.run(BaseCommand.java:51)at com.android.commands.am.Am.main(Am.java:122)at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:262)'; Code: '1' (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 349.53 secondsBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:14.666Z'System info: host: 'LAP-026', ip: '192.168.10.178', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_112'Driver info: driver.version: RemoteWebDriver]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.WebDriverException: An unknown server-side error occurred while processing the command. Original error: Error occured while starting App. Original error: Error executing adbExec. Original error: 'Command 'C\:\\Users\\salakhya\\AppData\\Local\\sdks\\platform-tools\\adb.exe -P 5037 -s ZY223ZKZ33 shell am start -W -n com.ubercab.eats/com.ubercab.eats.app.feature.launch.LobbyActivity -S' exited with code 1'; Stderr: 'java.lang.SecurityException: Permission Denial: starting Intent { flg=0x10000000 cmp=com.ubercab.eats/.app.feature.launch.LobbyActivity } from null (pid=17317, uid=2000) not exported from uid 10162at android.os.Parcel.readException(Parcel.java:1683)at android.os.Parcel.readException(Parcel.java:1636)at android.app.ActivityManagerProxy.startActivityAndWait(ActivityManagerNative.java:3266)at com.android.commands.am.Am.runStart(Am.java:634)at com.android.commands.am.Am.onRun(Am.java:390)at com.android.internal.os.BaseCommand.run(BaseCommand.java:51)at com.android.commands.am.Am.main(Am.java:122)at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:262)'; Code: '1' (WARNING: The server did not provide any stacktrace information)Command duration or timeout: 349.53 secondsBuild info: version: '3.8.1', revision: '6e95a6684b', time: '2017-12-01T19:05:14.666Z'System info: host: 'LAP-026', ip: '192.168.10.178', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_112'Driver info: driver.version: RemoteWebDriverat sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)at sun.reflect.NativeConstructorAccessorImpl.newInstance(Unknown Source)at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(Unknown Source)at java.lang.reflect.Constructor.newInstance(Unknown Source)at org.openqa.selenium.remote.ErrorHandler.createThrowable(ErrorHandler.java:214)at org.openqa.selenium.remote.ErrorHandler.throwIfResponseFailed(ErrorHandler.java:166)at org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$new$0(JsonWireProtocolResponse.java:53)at org.openqa.selenium.remote.JsonWireProtocolResponse.lambda$getResponseFunction$2(JsonWireProtocolResponse.java:91)at org.openqa.selenium.remote.ProtocolHandshake.lambda$createSession$0(ProtocolHandshake.java:123)at java.util.stream.ReferencePipeline$3$1.accept(Unknown Source)at java.util.Spliterators$ArraySpliterator.tryAdvance(Unknown Source)at java.util.stream.ReferencePipeline.forEachWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyIntoWithCancel(Unknown Source)at java.util.stream.AbstractPipeline.copyInto(Unknown Source)at java.util.stream.AbstractPipeline.wrapAndCopyInto(Unknown Source)at java.util.stream.FindOps$FindOp.evaluateSequential(Unknown Source)at java.util.stream.AbstractPipeline.evaluate(Unknown Source)at java.util.stream.ReferencePipeline.findFirst(Unknown Source)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:126)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:142)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:601)at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:219)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:142)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:155)at MobileApp.MobileApp.uber.setUp(uber.java:39)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)at java.lang.reflect.Method.invoke(Unknown Source)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:451)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)at org.testng.TestRunner.beforeRun(TestRunner.java:529)at org.testng.TestRunner.run(TestRunner.java:497)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.WebDriverException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- DeliveryLocation -->
        <test-method status="SKIP" signature="End()[pri:0, instance:MobileApp.MobileApp.uber@7e0e6aa2]" name="End" is-config="true" duration-ms="0" started-at="2018-03-14T15:24:30Z" finished-at="2018-03-14T15:24:30Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- End -->
      </class> <!-- MobileApp.MobileApp.uber -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
